src/mpi_constants.c

```diff
@@ -23,3 +23,5 @@ MPI_Datatype c_MPI_LOGICAL = MPI_LOGICAL;
 MPI_Datatype c_MPI_CHARACTER = MPI_CHARACTER;
 
 MPI_Datatype c_MPI_REAL = MPI_REAL;
+
+MPI_Comm c_MPI_COMM_NULL = MPI_COMM_NULL;
```

Comment:
Now that we've enough code in `mpi_constants.c`, we should probably refactor `mpi_constants.c` by grouping `MPI_Comm`'s together and simiarly the same for `MPI_Datatype`'s as well.

i.e. simply have:
```c
// all MPI_Datatype definitions/declarations go here
MPI_Datatype c_MPI_DOUBLE = MPI_DOUBLE;

MPI_Datatype c_MPI_FLOAT = MPI_FLOAT;

MPI_Datatype c_MPI_INT = MPI_INT;

...

// all MPI_Comm definition/declaration's go here

MPI_Comm c_MPI_COMM_WORLD = ...
...
...
```
```